src/shared/inpage-types.ts

```diff
@@ -3,12 +3,12 @@
  */
 export enum DomEventName {
   request = 'request',
-  authenticationRequest = 'stacksAuthenticationRequest',
-  signatureRequest = 'signatureRequest',
-  structuredDataSignatureRequest = 'structuredDataSignatureRequest',
-  transactionRequest = 'stacksTransactionRequest',
-  profileUpdateRequest = 'profileUpdateRequest',
-  psbtRequest = 'psbtRequest',
+  authenticationRequest = 'hiroWalletStacksAuthenticationRequest',
```

Comment:
Maybe we should use a `const` somewhere for `hiroWallet` so it will be easier to change this to `leatherWallet` ? 🤔